nsqadmin/static/js/views/channel.hbs

```diff
@@ -69,7 +69,7 @@
         {{#each nodes}}
         <tr>
             <td>
-                <a class="link" href="/nodes/{{node}}">{{node}}</a>
+                {{hostname}} (<a class="link" href="/nodes/{{node}}">{{node}}</a>)
```

Comment:
Can we apply similar logic that i added in the client connection section where we skip `hostname` if it's the value used in the broadcast address and therefore already in the `node` value here? Not having redundant values (ie: `nsq01.company.com (nsq01.company.com:4151)` would be nice.)